src/blueapi/client/client.py

```diff
@@ -415,7 +416,10 @@ def _wait_for_reload(
                 raise BlueskyRemoteControlError(
                     f"Error reloading environment: {status.error_message}"
                 )
-            elif status.initialized:
+            elif (
+                status.initialized is True
```

Comment:
Could: Was there a reason you needed this?

```suggestion
                status.initialized
```

Reply:
I agree that it will be initialized when a new environment with a different UUID is created, as the start method always initializes it to `True`. However, I think it doesn't hurt to have this here... 

if it was about being explicit with the `True` I did it because of this comment in the https://github.com/DiamondLightSource/blueapi/pull/764#discussion_r1890086964